README.md

```diff
@@ -339,6 +342,36 @@ async function myFunction() {
 }
 ```
 
+#### <a name="getGridForUser" />`Firestore.get.gridForUser(userId, gridId)`
+Gets the grid object for a logged in user. The userId corresponds to the user that is currently logged in. The gridId is the requested grid object. Note: all Firestore functions do this automatically.
+* <b>userId</b> `String` - The id of the logged in user.
```

Comment:
Maybe we can clarify somewhere that these are user email addresses?

Reply:
Oh I guess that's shown enough by example.

Reply:
Resolved and merged.